src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import slick.backend.DatabaseConfig
+import slick.driver.JdbcProfile
+import slick.driver.H2Driver.api._
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future}
+
+/**
+ * Created by dvoet on 2/3/16.
+ */
+trait TestDriverComponent extends DriverComponent {
+  override implicit val executionContext = scala.concurrent.ExecutionContext.Implicits.global
+
+  val databaseConfig: DatabaseConfig[JdbcProfile] = DatabaseConfig.forConfig[JdbcProfile]("h2mem1")
+  override val driver: JdbcProfile = databaseConfig.driver
+  val database = databaseConfig.db
+
+  protected def runAndWait[R](action: DBIOAction[R, _ <: NoStream, _ <: Effect], duration: Duration = Duration.Inf): R = {
```

Comment:
Inf makes me uncomfortable.  Could we make it a different unreasonably long duration, say 1 minute?